collection_app_2020-09-28.sql

```diff
@@ -0,0 +1,64 @@
+# ************************************************************
+# Sequel Pro SQL dump
+# Version 4541
+#
+# http://www.sequelpro.com/
+# https://github.com/sequelpro/sequelpro
+#
+# Host: 127.0.0.1 (MySQL 5.7.31)
+# Database: collection_app
+# Generation Time: 2020-09-28 10:20:57 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table collection_data
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `collection_data`;
+
+CREATE TABLE `collection_data` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `artist_firstname` varchar(100) NOT NULL DEFAULT '',
+  `artist_lastname` varchar(100) DEFAULT NULL,
+  `album` varchar(100) NOT NULL DEFAULT '',
+  `year` year(4) NOT NULL,
+  `cover_art` varchar(1000) DEFAULT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+LOCK TABLES `collection_data` WRITE;
+/*!40000 ALTER TABLE `collection_data` DISABLE KEYS */;
+
+INSERT INTO `collection_data` (`id`, `artist_firstname`, `artist_lastname`, `album`, `year`, `cover_art`)
```

Comment:
once you have changed the table name, having `artist_firstname` will be redundant and you can just have `firstname`